libs/f3/db/sql.php

```diff
@@ -461,8 +461,7 @@ function __construct($dsn,$user=NULL,$pw=NULL,array $options=NULL) {
 		if (!$options)
 			$options=array();
 		if (isset($parts[0]) && strstr($parts[0],':',TRUE)=='mysql')
-			$options+=array(\PDO::MYSQL_ATTR_INIT_COMMAND=>'SET NAMES '.
-				strtolower(str_replace('-','',$fw->get('ENCODING'))).';');
+			$options+=array(\PDO::MYSQL_ATTR_INIT_COMMAND=>'SET NAMES utf8mb4;');
```

Comment:
OK, thanks. I created the following issue, but I don't think I can help more on this problem.

https://github.com/bcosca/fatfree/issues/976